lib/core/src/Cardano/Wallet/Api/Client.hs

```diff
@@ -333,7 +337,9 @@ transactionClient =
             , postTransactionFee = _postTransactionFee
             , postExternalTransaction = _postExternalTransaction . fromSerialisedTx
             , deleteTransaction = _deleteTransaction
-            , getTransaction = _getTransaction
+            , getTransaction =
+                \wid txid metadataSchema ->
+                _getTransaction wid txid (toSimpleMetadataFlag metadataSchema)
```

Comment:
I wasn't quite sure about the `do` notation in other places, but I also have the feeling here that the body of the lambda should be indented relative to its head.

Reply:
yep, this is why I really would like to have an autoformatter. I'm not indenting anything by hand since quite some time